Embedded System L1/AVR ATMEGA32/DIO/PROJECT/DIO_program.c

```diff
@@ -0,0 +1,109 @@
+/******************************************************************************************************
+ * File: DIO_program.c 
+ * Author: Khalid Mosaad Zakaria
+ * Data: 10 july 2021
+ * Description: This File include Definitions of functions and Strucrures.
+*******************************************************************************************************/
+
+#include"STD_TYPES.h"
+#include"BIT_MATH.h"
+#include"DIO_register.h"
+#include"DIO_interface.h"
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of c files

[Sprints_CodeGuideLine.pdf](https://github.com/KhalidMosad/Sprints-Tasks/files/6901176/Sprints_CodeGuideLine.pdf)

![4](https://user-images.githubusercontent.com/44152298/127508098-dd2316ae-1851-429c-80c2-ae418e29dd28.PNG)